### PR TITLE
shipit_uplift: Remove coverage info for removed lines (as it was unused)

### PR DESCRIPTION
**Pull request by marco-c**
And add overall information to "/coverage/changeset" endpoint.

### CODE REVIEW

File: src/shipit_uplift/shipit_uplift/coverage.py [hunk 1]
```diff
@@ -203,50 +203,32 @@ def get_directory_coverage(changeset, prev_changeset, directory):
 coverage_service = CodecovCoverage()
 
 
-def get_coverage_builds(changeset, before=True, after=True):
+def get_coverage_build(changeset):
```
Comment (armenzg) [inline src/shipit_uplift/shipit_uplift/coverage.py]: The refactoring of this function looks great!

File: src/shipit_uplift/shipit_uplift/coverage_by_changeset_impl.py [hunk 2]
```diff
@@ -30,39 +30,30 @@ def generate(changeset):
             continue
 
         # Retrieve coverage of removed and added lines.
-        new_coverage = coverage_service.get_file_coverage(next_build_changeset, new_path)
-        old_coverage = coverage_service.get_file_coverage(previous_build_changeset, old_path)
+        coverage = coverage_service.get_file_coverage(build_changeset, new_path)
```
Comment (armenzg) [inline src/shipit_uplift/shipit_uplift/coverage_by_changeset_impl.py]: Please update the comment.

File: src/shipit_uplift/shipit_uplift/coverage_by_changeset_impl.py [hunk 3]
```diff
@@ -73,7 +64,8 @@ def generate(changeset):
         })
 
     return {
-        'build_changeset': next_build_changeset,
-        'previous_build_changeset': previous_build_changeset,
+        'build_changeset': build_changeset,
+        'overall_cur': overall['cur'],
+        'overall_prev': overall['prev'],
```
Comment (armenzg) [inline src/shipit_uplift/shipit_uplift/coverage_by_changeset_impl.py]: 👍

File: src/shipit_uplift/shipit_uplift/coverage_by_changeset_impl.py (hunk 2 above)
Comment (marco-c) [reply src/shipit_uplift/shipit_uplift/coverage_by_changeset_impl.py]: Done!

**Pull request merged**